webhooks.md

```diff
@@ -2,6 +2,139 @@
 
 When an action of particular type is performed in the system, a message is sent to the configured webhook, using HTTP POST and JSON body.
 
+## General message
+
+```javascript
+{
+    "EnterpriseId": "851df8c8-90f2-4c4a-8e01-a4fc46b25178",
+    "IntegrationId": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+    "Events": [
+        {
+            "Discriminator": "ServiceOrderUpdated",
+            "Value": {
+                "Id": "bfee2c44-1f84-4326-a862-5289598f6e2d"
+            }
+        },
+        {
```

Comment:
I'd remove this one because it might make people feel that enterprise and integration will always be present. Let's go with simple example of just ServiceOrderUpdated.